### DISCUSSION TITLE
Error while Calibrating

### DISCUSSION THREAD

**Discussion by ghost** [iiasa/CWatM#150] [Q&A]
Hello!

Does anyone know how to resolve this error?

![image](https://github.com/user-attachments/assets/4e32f6a8-9191-414c-adda-ebf2d9e9691f)

This is what the log files say:
![image](https://github.com/user-attachments/assets/6e98215b-b678-49a5-97ee-7b3a1048d066)

**Comment by mikhailsmilovic** [comment]
Guten Morgen, in **settings_calibration.txt**, is
### bestrun = True
If not, would you set this equal to True and re-run the calibration? It will start where it left off.
